### PR TITLE
WIP Test e2e

### PR DESCRIPTION
**Pull request by slintes**
WIP

testing  peer check e2e

### CODE REVIEW

File: e2e/self_node_remediation_test.go [hunk 1]
```diff
@@ -535,7 +542,7 @@ func checkSnrLogs(node *v1.Node, expected []string) {
 
 	EventuallyWithOffset(1, func() string {
 		var err error
-		logs, err := utils.GetLogs(k8sClientSet, pod)
+		logs, err := utils.GetLogs(k8sClientSet, pod, since)
```
Comment (clobrano) [inline e2e/self_node_remediation_test.go:545]: nice!

File: e2e/self_node_remediation_test.go [hunk 2]
```diff
@@ -75,11 +75,12 @@ var _ = Describe("Self Node Remediation E2E", func() {
 
 		AfterEach(func() {
 			// restart snr pods for resetting logs...
-			restartSnrPods(workers)
+			//restartSnrPods(workers)
 		})
 
 		JustAfterEach(func() {
-			printSNRLogsFromNode(&workers.Items[1])
+			// TODO why worker 1??
```
Comment (clobrano) [inline e2e/self_node_remediation_test.go:82]: No idea, it must be a mistake

Comment (clobrano) [reply e2e/self_node_remediation_test.go:82]: ah no, it comes from a "refactory", originally it was the log of `healthyNode := &workers.Items[1]`

Comment (clobrano) [reply e2e/self_node_remediation_test.go:82]: I could've kept at least the variable name to make it clearer

**Comment by slintes** [comment]
closing in favour of #226 

/close

**Pull request closed without merging**